service/project.go

```diff
@@ -630,6 +631,7 @@ func (uis *UIServer) modifyProject(w http.ResponseWriter, r *http.Request) {
 	projectRef.DisabledStatsCache = &responseRef.DisabledStatsCache
 	projectRef.PeriodicBuilds = []model.PeriodicBuildDefinition{}
 	projectRef.PerfEnabled = &responseRef.PerfEnabled
+	projectRef.ContainerSizes = responseRef.ContainerSizes
```

Comment:
Should there be validation above this like there is for the other structs to make sure the container sizes are valid?